web/src/settings_panel_menu.js

```diff
@@ -151,7 +151,10 @@ export class SettingsPanelMenu {
         this.$main_elem.children("li").removeClass("active");
         this.$curr_li.addClass("active");
 
-        const settings_section_hash = "#" + this.hash_prefix + section;
+        const settings_section_hash =
+            section === "user-list-admin"
+                ? `#${this.hash_prefix}${section}/active-users-admin`
```

Comment:
The browser back button is not working correctly because the hash gets changed from `http://localhost:9991/#organization/user-list-admin/deactivated-users-admin` to `http://localhost:9991/#organization/user-list-admin/active-users-admin` here.

Reply:
Okay , so I have updated the code to update the hash for `user-list-admin` when the hash is set for each section based on data-section attribute ; i.e even before the activate_section_or_default function is called.
So this does not update the hash to `#organization/user-list-admin/active-users-admin` everytime the back button is clicked.

Though this works as we expect on pressing the back button , the section rendered doesn't change even though the url does change. Is this behaviour explainable? 
If so , should the probable solution be to always use the .goto() function for rendering a sub-section ?

Reply:
Can you update the PR with the changes you have done? I can take a look and we could debug what is preventing the UI to change.

Reply:
Sure,  I have updated the PR.